### PR TITLE
BZ#1871800 - Use external facing NTP

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1871800

Preview: https://bz-1871800--ocpdocs.netlify.app/openshift-enterprise/latest/installing/install_config/installing-customizing.html#installation-special-config-crony_installing-customizing

### CODE REVIEW

**Comment by jboxman** [comment]
@mike-nguyen, can you confirm if this change makes sense or suggest an alternate approach?

Thanks!

**Comment by jboxman** [comment]
@sferich888, does this change make sense? The current server is unreachable.

**Review by sferich888 — COMMENTED** [review]
Why can't a customer specify any valid time server?

**Comment by jboxman** [comment]
@sferich888, can they? I'm not that familiar with this part of the docs. I can apply whatever fix makes the most sense here, I just want to ensure we aren't suggesting an unreachable host.

Thanks.

**Comment by sferich888** [comment]
The bug suggests were changing this to better document what redhat clock sources to use. So the changes make since, but in theory a customer could use any clock source (so long ase you can connect and it's valid).

**Comment by jboxman** [comment]
Refs https://github.com/openshift/openshift-docs/pull/25764.

File: modules/installation-special-config-crony.adoc [hunk 1]
```diff
@@ -16,13 +16,14 @@ to your nodes as a MachineConfig.
...
 $ cat << EOF | base64
-    server clock.redhat.com iburst
+    pool 0.rhel.pool.ntp.org iburst <1>
     driftfile /var/lib/chrony/drift
     makestep 1.0 3
     rtcsync
     logdir /var/log/chrony
     EOF
 ----
+<1> Specify any valid, reachable time source, or you can specify any of the following NTP servers: `1.rhel.pool.ntp.org`, `2.rhel.pool.ntp.org`, or `3.rhel.pool.ntp.org`.
```
Comment (jeana-redhat) [inline modules/installation-special-config-crony.adoc:26]: Would suggest breaking this into two sentences. Maybe:
Specify any valid, reachable time source. Alternatively, you can specify any of the following NTP servers: `1.rhel.pool.ntp.org`, `2.rhel.pool.ntp.org`, or `3.rhel.pool.ntp.org`.

**Review by jeana-redhat — APPROVED** [review]
Small suggestion, LGTM though :)

File: modules/installation-special-config-crony.adoc (hunk 1 above)
Comment (jboxman) [reply modules/installation-special-config-crony.adoc:26]: @jeana-redhat thanks!

**Pull request merged**